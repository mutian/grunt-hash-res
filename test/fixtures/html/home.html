<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<title>test</title>
	<link rel="stylesheet" href="//s.cctcdn.cn/w/1611/css/test.css">
	<link rel="stylesheet" href="//s.cctcdn.cn/w/1611/css/pages/home.css">
</head>
<body>
	<img src="//s.cctcdn.cn/w/1611/images/global/logo.png?type=2" alt="test">
	<img src="//s.cctcdn.cn/w/1611/images/blank.png" alt="test">
</body>
</html>
