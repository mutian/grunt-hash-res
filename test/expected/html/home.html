<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<title>test</title>
	<link rel="stylesheet" href="//s.cctcdn.cn/w/1611/css/test.css?v=b3309ed">
	<link rel="stylesheet" href="//s.cctcdn.cn/w/1611/css/pages/home.css?v=b3309ed">
</head>
<body>
	<img src="//s.cctcdn.cn/w/1611/images/global/logo.png?v=a1a20a7&type=2" alt="test">
	<img src="//s.cctcdn.cn/w/1611/images/blank.png?v=7ba2edd" alt="test">
</body>
</html>
